### ISSUE TITLE
Endless repetition of log messages that take up almost exactly one line

### ISSUE THREAD

**Issue by ToxicFrog** [moon-chilled/slashem9#5]
![image](https://user-images.githubusercontent.com/90456/76879350-823c1400-684c-11ea-97be-c28db22c68b8.png)

This seems to happen primarily with "$shopkeeper offers $money for your $item. Do you want to sell it?" and "You have a little trouble lifting the $item" messages; an easy way to reproduce is get burdened and then run around picking stuff up until you grab something with exactly the right name length. Note that all of those messages appeared instantly as soon as I chose to drop the mythril coat, and once this happens it won't respond to `ynaq` to select an answer.

You can escape from the endless message spam by hitting `esc` a few times, which is usually safe if you're being prompted; resizing the terminal or turning off `perm_invent` temporarily are also valid workarounds.

**Comment by moon-chilled** [comment]
Is this fixed with https://github.com/ToxicFrog/slashem9/commit/d5b15eee399485fce234b29e4a31ff1e88106b9c (the `linespace` tweak)?

**Comment by ToxicFrog** [comment]
Nope. I didn't really expect it to; the `linespace` change produces the same result where `border_space == 1` (which is the case here). The change is just so that it will continue to do the right thing if `border_space` changes.

I found the `>>` fix while I was in there poking around trying to find a fix for this, but no luck as yet.

I came up with what seems to be a decently reliable repro, though: roll wizard (or some other class that starts with a bunch of items with relatively long names, like spellbooks), then stand on a staircase and drop your whole inventory, likely triggering the "the %s hits another object and falls down the stairs!" message.
